modules/core/src/utils/typed-array-manager.js

```diff
@@ -9,7 +9,11 @@ export class TypedArrayManager {
     Object.assign(this.props, props);
   }
 
-  allocate(typedArray, count, {size = 1, type, padding = 0, copy = false, initialize = false}) {
+  allocate(
+    typedArray,
+    count,
+    {size = 1, type, padding = 0, copy = false, initialize = false, overAlloc}
```

Comment:
Fine, though I am not sure we need this much control of over alloc. I think we could just cap the amount of memory we over allocate.